pkg/extensions/search/cve/trivy/scanner.go

```diff
@@ -221,7 +221,7 @@ func (scanner Scanner) IsImageFormatScannable(image string) (bool, error) {
 		}
 	}
 
-	return false, nil
+	return true, nil
```

Comment:
Behavior in this func is changing, in what way?

Reply:
Because in the previous version only the first layer was checked for compatibility. Both cases were returning from the function on the first iteration of the for loop.

Reply:
Previous version of line 224 wasn't even reached, and I noticed while checking coverage.